lib/modules/datasource/github-runners/index.spec.ts

```diff
@@ -3,18 +3,58 @@ import { GithubRunnersDatasource } from '.';
 
 describe('modules/datasource/github-runners/index', () => {
   describe('getReleases', () => {
-    it('returns releases if package is known', async () => {
+    it('returns releases for ubuntu', async () => {
       const res = await getPkgReleases({
         datasource: GithubRunnersDatasource.id,
         packageName: 'ubuntu',
       });
 
       expect(res).toMatchObject({
         releases: [
-          { version: '18.04' },
+          { version: '18.04', isDeprecated: true },
           { version: '20.04' },
           { version: '22.04' },
         ],
+        sourceUrl: 'https://github.com/actions/runner-images',
+      });
+    });
+
+    it('returns releases for macos', async () => {
```

Comment:
```suggestion
    it('returns releases for macOS', async () => {
```

We should capitalize the same as the official name.